File: TeamCode/src/main/java/robotuprising/ftc2021/opmodes/examples/exOpMode.java
package robotuprising.ftc2021.opmodes.examples;

import com.qualcomm.robotcore.eventloop.opmode.OpMode;

public class exOpMode extends OpMode {

    private exIntake bIntake;

    @Override
    public void init() {
        bIntake = new exIntake(hardwareMap);
    }

    @Override
    public void loop() {
        bIntake.turnIntakeOn();
        bIntake.getDistanceSensorReading();
    }
}

File: TeamCode/src/main/java/robotuprising/ftc2021/opmodes/examples/exOpMode.kt
package robotuprising.ftc2021.opmodes.examples

import com.qualcomm.robotcore.eventloop.opmode.OpMode
import robotuprising.ftc2021.opmodes.examples.exIntake

class exOpMode : OpMode() {
    private var bIntake: exIntake? = null
    override fun init() {
        bIntake = exIntake(hardwareMap)
    }

    override fun loop() {
        bIntake!!.turnIntakeOn()
        bIntake!!.distanceSensorReading
    }
}